extensions/interactions/baseValidatorSpec.js

```diff
@@ -30,13 +30,17 @@ describe('Interaction validator', function() {
   var currentState, otherState, goodOutcomeDest, goodOutcomeFeedback;
   var badOutcome, goodAnswerGroups, goodDefaultOutcome;
 
-  var createAnswerGroup = function(outcome, ruleSpecs) {
+  var createAnswerGroup = function(outcome, ruleSpecs, correct) {
     if (!ruleSpecs) {
       ruleSpecs = [];
     }
+    if (!correct) {
```

Comment:
A cleaner way to do this would be to use `!!correct`, which would cast `correct` to bool. You could do this directly in the return statement, with
```
  return {
    ruleSpecs: ruleSpecs,
    outcome: outcome,
    correct: !!correct
  };
```